src/test/java/DecideTest.java

```diff
@@ -55,6 +55,52 @@ public void LIC0Exception () {
 
     // LIC1
 
+    /**
+     * Check that LIC1 returns true if there exists at least one set of three consecutive data points
+     * that cannot all be contained within or on a circle of radius RADIUS1.
+     */
+    @Test
+    public void LIC1PositiveCase() throws IllegalParameterObjectException {
+        double[][] points = new double[][]{{0.0, 0.0}, {0.0, 0.0}, {-10.0, -10.0}, {10.0, 10.0}};
+        int numpoints = points.length;
+        Parameters parameters = new Parameters(0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0);
+        Decide decide = new Decide(numpoints, points, parameters, (LCM)null, (boolean[])null);
+        boolean LIC1True = decide.LIC1();
+        assertTrue(LIC1True);
+    }
+
+    /**
+     * Check that LIC1 returns false if there does not exist at least one set of three consecutive data points
+     * that cannot all be contained within or on a circle of radius RADIUS1.
+     */
+    @Test
+    public void LIC1NegativeCase() throws IllegalParameterObjectException {
+        double[][] points = new double[][]{{0.0, 0.0}, {0.0, 0.0}, {0.0, 1.0}, {1.0, 0.0}};
+        int numpoints = points.length;
+        Parameters parameters = new Parameters(0, 5, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0);
+        Decide decide = new Decide(numpoints, points, parameters, (LCM)null, (boolean[])null);
+        boolean LIC1False = decide.LIC1();
+        assertFalse(LIC1False);
+    }
```

Comment:
These tests are flipped. The points in the positive one cannot be contained within the circle and the points in the negative one can. I think they pass because as of now the LIC1 function checks if the points are outside of the circle. I mentioned this in other comments as well.

Reply:
I think that the tests are correct, from how I have understood the description of LIC1.

Reply:
However, I wonder if we should have a test with a triangle that has an area greater than 0? Since we do not, we do not test the code on rows 65-69.